Userland/Shell/CMakeLists.txt

```diff
@@ -4,11 +4,21 @@ serenity_component(
     TARGETS Shell
 )
 
+serenity_component(
+    ShellMinimal
+    REQUIRED
+    TARGETS ShellMinimal
+)
+
 if (ENABLE_FUZZERS)
     add_custom_target(Shell) # Stub target to placate serenity_component
+    add_custom_target(ShellMinimal) # Stub target to placate serenity_component
 else()
     set(SOURCES main.cpp)
 
     serenity_bin(Shell)
     target_link_libraries(Shell PRIVATE LibCore LibShell LibFileSystem LibLine LibMain)
+
+    serenity_bin(ShellMinimal)
```

Comment:
uhh...but why?
Nothing uses this, and from the user's point of view this and the regular shell are the exact same